views/templates/front/get-started.tpl

```diff
@@ -0,0 +1,38 @@
+{*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen PrestaShop plugin
+ *
+ * @author Adyen BV <support@adyen.com>
+ * @copyright (c) 2020 Adyen B.V.
+ * @license https://opensource.org/licenses/MIT MIT license
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *}
+
+<div style="display: flex;">
+    <div style="float:left;padding-right:10px;">
+        <img width="120px" src="{$logo}">
+    </div>
+    <div>
+        <p>Adyen all-in-one payments platform.</p>
+        <p>Offer key payment methods anywhere in the world at the flick of a switch.</p>
+        <p>Easy integration with our in-house built PrestaShop Plugin, no set-up fee.</p>
```

Comment:
Let's explain they need an account on Adyen. They can create this account on  https://www.adyen.com/signup

Reply:
Added the signup link with the wording used in most documentation (https://docs.adyen.com/classic-integration/3d-secure/3d-secure-1/3d-secure-1-authentication-only)